README.md

```diff
@@ -1378,28 +1449,25 @@ worth mentioning a few other flavors of equity compensation out there.
 
 ## Tax Basics
 
-When you are awarded equity compensation, you typically also have to pay taxes.
-In fact, depending on your situation, equity compensation can give rise to multiple kinds
-of taxes, including federal and state income taxes as well as employment taxes and
-Medicare surtax charges.
-As we’ll see, sometimes you need to pay at different times, and even when you don’t have
-the cash.
+When you are awarded equity compensation, you have to consider the tax consequences.
+Depending on your situation, equity compensation can give rise to multiple kinds of taxes,
+including federal and state income taxes as well as employment taxes.
+
+[Skip ahead](#taxes-on-equity-compensation) to understand how taxes on equity work, but if you
+have time, this section gives a fast and detailed overview of tax fundamentals.
```

Comment:
Good changes here, but I urge us not to assign expected lengths of time ("fast") to get through any sections. If this is new material to some readers, they may not get through it that quickly! (I will add this to Manual). Elide "a fast and," as "overview" implies that it will not be too lengthy in any case.

Reply:
Rewriting.